Mathlib/Topology/Category/Profinite/Basic.lean

```diff
@@ -250,4 +250,7 @@ theorem epi_iff_surjective {X Y : Profinite.{u}} (f : X ⟶ Y) : Epi f ↔ Funct
   · rw [← CategoryTheory.epi_iff_surjective]
     apply (forget Profinite).epi_of_epi_map
 
+/-- The product space of profinite spaces is profinite. -/
```

Comment:
Not sure "product" is the right term; dependent functions, or pi-type is more appropriate.

Reply:
Changed into pi-type